src/components/SelectTypeQuestion/SelectTypeQuestion.tsx

```diff
@@ -12,35 +12,33 @@ interface Selected {
 }
 
 function SelectTypeQuestion() {
-  const { questionResponse, state, quizNextHandler, quizBackHandler, isFirstQuestion } =
-    useContext(QuizContext);
+  const { state, getNextQuestion, getPreviousQuestion } = useContext(QuizContext);
   let question;
   let type: `${QuestionTypes}`;
   let hasImages = false;
 
-  if (questionResponse) {
-    question = questionResponse.next_question;
+  if (state?.quiz.currentQuestion) {
```

Comment:
I left it on purpose so I know that it can't be undefined. because what gets returned from useQuiz in state is populated by me and I want it to always have a `state.quiz`. 

Usually I avoid adding optional chaining while I'm using typescript if not necessary so if a certain value I don't expect it to be undefined typescript would warn me about it. What do you think?

Reply:
works for me. so in this case shall we switch from:
`state?.quiz.currentQuestion` to `state.quiz.currentQuestion`?